Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md

```diff
@@ -38,9 +38,12 @@ Open powershell (in admin mode) and run the following commands. When prompted fo
 Requires the application instance ID created above. Continue your powershell session and run the following commands.
   * `New-CsTeamsComplianceRecordingPolicy -Enabled $true -Description "Test policy created by <yourName>" <policyIdentity>`
   * ```Set-CsTeamsComplianceRecordingPolicy -Identity <policyIdentity> -ComplianceRecordingApplications ` @(New-CsTeamsComplianceRecordingApplication -Parent <policyIdentity> -Id <objectId>)```
```

Comment:
I am curious why this step alone did not associate the app with the policy. Documentation below seems to suggest that it should 
https://learn.microsoft.com/en-us/powershell/module/teams/set-csteamscompliancerecordingpolicy?view=teams-ps